builder/amazon/common/run_config.go

```diff
@@ -14,6 +14,16 @@ import (
 
 var reShutdownBehavior = regexp.MustCompile("^(stop|terminate)$")
 
+type AmiFilterOptions struct {
+	Filters    map[*string]*string
```

Comment:
Because amazon wants `*string` and you can't easily do `&"foo"` without assigning it to a temp variable.